appinfo/info.xml

```diff
@@ -16,7 +16,7 @@ And in the works for the [coming versions](https://github.com/nextcloud/spreed/m
 
 	]]></description>
 
-	<version>8.0.5</version>
+	<version>8.0.5.1</version>
```

Comment:
[compatibility test fails](https://drone.nextcloud.com/nextcloud/spreed/6085/1/2), but update works fine and it should go away with 8.0.6, so it should not be an issue.